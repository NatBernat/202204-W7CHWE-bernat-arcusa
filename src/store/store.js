import { configureStore } from "@reduxjs/toolkit";
import { usersReducer } from "../redux/features/usersSlice";

const store = configureStore({
  reducer: {
    users: usersReducer,
  },
});

export default store;
