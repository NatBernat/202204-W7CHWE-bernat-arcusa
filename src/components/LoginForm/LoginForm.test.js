import { Provider } from "react-redux";
import TestRenderer from "react-test-renderer";
import store from "../../store/store";
import LoginForm from "./LoginForm";

describe("Given a loginForm component", () => {
  describe("When it's rendered", () => {
    test("Then it should always match this snapshot", () => {
      const renderedLoginForm = TestRenderer.create(
        <Provider store={store}>
          <LoginForm />
        </Provider>
      );

      expect(renderedLoginForm).toMatchSnapshot();
    });
  });
});
