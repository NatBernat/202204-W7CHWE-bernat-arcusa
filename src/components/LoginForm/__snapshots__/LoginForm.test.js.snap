// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given a loginForm component When it's rendered Then it should always match this snapshot 1`] = `
<form
  autoComplete="off"
  className="sc-bczRLJ hyAoSz"
  noValidate={true}
  onSubmit={[Function]}
>
  <label
    htmlFor="username"
  />
  <input
    id="username"
    onChange={[Function]}
    placeholder="Username"
    value=""
  />
  <label
    htmlFor="password"
  />
  <input
    id="password"
    onChange={[Function]}
    placeholder="Password"
    type="password"
    value=""
  />
  <button
    disabled={true}
    type="submit"
  >
    Login
  </button>
</form>
`;
